<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>307</width>
    <height>461</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>计算器-蔡绮岚-2022414040101</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
	min-width: 140px;
	min-height: 300px;
	margin: 20px;
	background-color: #000000;
}

QLineEdit {
	font-size: 34px;
	border-radius: 2px;
	padding: 5px;
	background-color: #000000;
	color: #ffffff; 
	height: 100px;
}

QPushButton {
	background-color: #2a2a2c;
	font-size: 20px;
	padding: 0px;
	margin : 0px;
	border-radius: 0px;
	color: #ffffff; 
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>3</number>
    </property>
    <property name="leftMargin">
     <number>1</number>
    </property>
    <property name="topMargin">
     <number>1</number>
    </property>
    <property name="rightMargin">
     <number>1</number>
    </property>
    <property name="bottomMargin">
     <number>1</number>
    </property>
    <item>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="0">
       <widget class="QPushButton" name="btnInverse">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #5c5c5e; 
}
QPushButton:hover {
    background-color: #8c8c8c; /* 鼠标悬停时的背景颜色 */
}
QPushButton:pressed {
    background-color: #8c8c8c; /* 按钮被按下时的背景颜色 */
	color: #e2e2e2;
}</string>
        </property>
        <property name="text">
         <string>1/x </string>
        </property>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="QPushButton" name="btnPlus">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #fea00c; 
	font-size: 24px;
}
QPushButton:hover {
    background-color: #fbc78e; /* 鼠标悬停时的背景颜色 */
}
QPushButton:pressed {
    background-color: #fbc78e; /* 按钮被按下时的背景颜色 */
	color: #fcf0e6;
}</string>
        </property>
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QPushButton" name="btnPeriod">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton:hover {
    background-color: #727272; /* 鼠标悬停时的背景颜色 */
}
QPushButton:pressed {
    background-color: #727272; /* 按钮被按下时的背景颜色 */
    color: #d0d0d0;
}
</string>
        </property>
        <property name="text">
         <string>.</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QPushButton" name="btnNum3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton:hover {
    background-color: #727272; /* 鼠标悬停时的背景颜色 */
}
QPushButton:pressed {
    background-color: #727272; /* 按钮被按下时的背景颜色 */
    color: #d0d0d0;
}
</string>
        </property>
        <property name="text">
         <string>3</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QPushButton" name="btnMultiple">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #fea00c; 
	font-size: 24px;
}
QPushButton:hover {
    background-color: #fbc78e; /* 鼠标悬停时的背景颜色 */
}
QPushButton:pressed {
    background-color: #fbc78e; /* 按钮被按下时的背景颜色 */
	color: #fcf0e6;
}</string>
        </property>
        <property name="text">
         <string>×</string>
        </property>
       </widget>
      </item>
      <item row="6" column="3">
       <widget class="QPushButton" name="btnEqual">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #fea00c; 
	font-size: 24px;
}
QPushButton:hover {
    background-color: #fbc78e; /* 鼠标悬停时的背景颜色 */
}
QPushButton:pressed {
    background-color: #fbc78e; /* 按钮被按下时的背景颜色 */
	color: #fcf0e6;
}</string>
        </property>
        <property name="text">
         <string>=</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="btnClearEntry">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #5c5c5e; 
}
QPushButton:hover {
    background-color: #8c8c8c; /* 鼠标悬停时的背景颜色 */
}
QPushButton:pressed {
    background-color: #8c8c8c; /* 按钮被按下时的背景颜色 */
	color: #e2e2e2;
}</string>
        </property>
        <property name="text">
         <string>CE</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="btnNum9">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton:hover {
    background-color: #727272; /* 鼠标悬停时的背景颜色 */
}
QPushButton:pressed {
    background-color: #727272; /* 按钮被按下时的背景颜色 */
    color: #d0d0d0;
}
</string>
        </property>
        <property name="text">
         <string>9</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QPushButton" name="btnNum0">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton:hover {
    background-color: #727272; /* 鼠标悬停时的背景颜色 */
}
QPushButton:pressed {
    background-color: #727272; /* 按钮被按下时的背景颜色 */
    color: #d0d0d0;
}
</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="btnNum8">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton:hover {
    background-color: #727272; /* 鼠标悬停时的背景颜色 */
}
QPushButton:pressed {
    background-color: #727272; /* 按钮被按下时的背景颜色 */
    color: #d0d0d0;
}
</string>
        </property>
        <property name="text">
         <string>8</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QPushButton" name="btnSign">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton:hover {
    background-color: #727272; /* 鼠标悬停时的背景颜色 */
}
QPushButton:pressed {
    background-color: #727272; /* 按钮被按下时的背景颜色 */
    color: #d0d0d0;
}
</string>
        </property>
        <property name="text">
         <string>±</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QPushButton" name="btnDivide">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #fea00c; 
	font-size: 24px;
}
QPushButton:hover {
    background-color: #fbc78e; /* 鼠标悬停时的背景颜色 */
}
QPushButton:pressed {
    background-color: #fbc78e; /* 按钮被按下时的背景颜色 */
	color: #fcf0e6;
}</string>
        </property>
        <property name="text">
         <string>÷</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="btnClear">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #5c5c5e; 
}
QPushButton:hover {
    background-color: #8c8c8c; /* 鼠标悬停时的背景颜色 */
}
QPushButton:pressed {
    background-color: #8c8c8c; /* 按钮被按下时的背景颜色 */
	color: #e2e2e2;
}</string>
        </property>
        <property name="text">
         <string>C</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QPushButton" name="btnNum5">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton:hover {
    background-color: #727272; /* 鼠标悬停时的背景颜色 */
}
QPushButton:pressed {
    background-color: #727272; /* 按钮被按下时的背景颜色 */
    color: #d0d0d0;
}
</string>
        </property>
        <property name="text">
         <string>5</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="btnDel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #5c5c5e; 
}
QPushButton:hover {
    background-color: #8c8c8c; /* 鼠标悬停时的背景颜色 */
}
QPushButton:pressed {
    background-color: #8c8c8c; /* 按钮被按下时的背景颜色 */
	color: #e2e2e2;
}</string>
        </property>
        <property name="text">
         <string>⌫</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="btnPercnetge">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #5c5c5e; 
}
QPushButton:hover {
    background-color: #8c8c8c; /* 鼠标悬停时的背景颜色 */
}
QPushButton:pressed {
    background-color: #8c8c8c; /* 按钮被按下时的背景颜色 */
	color: #e2e2e2;
}</string>
        </property>
        <property name="text">
         <string>%</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="btnSqrt">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #5c5c5e; 
}
QPushButton:hover {
    background-color: #8c8c8c; /* 鼠标悬停时的背景颜色 */
}
QPushButton:pressed {
    background-color: #8c8c8c; /* 按钮被按下时的背景颜色 */
	color: #e2e2e2;
}</string>
        </property>
        <property name="text">
         <string>√x</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="btnSquare">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #5c5c5e; 
}
QPushButton:hover {
    background-color: #8c8c8c; /* 鼠标悬停时的背景颜色 */
}
QPushButton:pressed {
    background-color: #8c8c8c; /* 按钮被按下时的背景颜色 */
	color: #e2e2e2;
}</string>
        </property>
        <property name="text">
         <string>x^2</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QPushButton" name="btnNum4">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton:hover {
    background-color: #727272; /* 鼠标悬停时的背景颜色 */
}
QPushButton:pressed {
    background-color: #727272; /* 按钮被按下时的背景颜色 */
    color: #d0d0d0;
}
</string>
        </property>
        <property name="text">
         <string>4</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="btnNum7">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton:hover {
    background-color: #727272; /* 鼠标悬停时的背景颜色 */
}
QPushButton:pressed {
    background-color: #727272; /* 按钮被按下时的背景颜色 */
    color: #d0d0d0;
}
</string>
        </property>
        <property name="text">
         <string>7</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QPushButton" name="btnNum1">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton:hover {
    background-color: #727272; /* 鼠标悬停时的背景颜色 */
}
QPushButton:pressed {
    background-color: #727272; /* 按钮被按下时的背景颜色 */
    color: #d0d0d0;
}
</string>
        </property>
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QPushButton" name="btnNum2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton:hover {
    background-color: #727272; /* 鼠标悬停时的背景颜色 */
}
QPushButton:pressed {
    background-color: #727272; /* 按钮被按下时的背景颜色 */
    color: #d0d0d0;
}
</string>
        </property>
        <property name="text">
         <string>2</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QPushButton" name="btnNum6">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton:hover {
    background-color: #727272; /* 鼠标悬停时的背景颜色 */
}
QPushButton:pressed {
    background-color: #727272; /* 按钮被按下时的背景颜色 */
    color: #d0d0d0;
}
</string>
        </property>
        <property name="text">
         <string>6</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QPushButton" name="btnMinus">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #fea00c; 
	font-size: 32px;
}
QPushButton:hover {
    background-color: #fbc78e; /* 鼠标悬停时的背景颜色 */
}
QPushButton:pressed {
    background-color: #fbc78e; /* 按钮被按下时的背景颜色 */
	color: #fcf0e6;
}</string>
        </property>
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="4">
       <widget class="QLineEdit" name="display">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignBottom|Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing</set>
        </property>
        <property name="placeholderText">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>307</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
